src/Models/Team.php

```
@@ -55,6 +55,21 @@ public static function findByOrgId(Database $db, int $orgId): array
         return $stmt->fetchAll();
     }
 
+    /**
+     * Return formal team names for an organisation.
+     *
+     * @return string[]
+     */
+    public static function namesByOrgId(Database $db, int $orgId): array
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)